files/restore_backup.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
+import os
+import re
+import json
+import shutil
+
+DOCUMENTATION = '''
+---
+module: restore_backup
+short_description: restores a backup created with 'backup=true' using, for instance, the lineinfile module
+description:
+  - looks for a backup file in the same dir as the file to be restored and renames it to the original.
+author: Yoram Hekma
+version_added: 2.0
+requirements:
+  - python > 2.6
```

Comment:
This code does not actually require `python > 2.6`.  It appears to work on py24 as well.